apps/src/code-studio/pd/foorm/editor/components/FoormEntityEditor.jsx

```diff
@@ -37,6 +37,7 @@ class FoormEntityEditor extends React.Component {
     this.state = {
       livePreviewStatus: PREVIEW_ON,
       num_facilitators: 2,
+      workshop_id: 1,
```

Comment:
would it be much harder to default to zero or null instead? this is confusing (at least to me) because `1` looks like a valid workshop id. not sure if the users of this feature will find it confusing though.

Reply:
This is just for the Preview Area of the Foorm Editor. It just gives a value for testing with. Will not impact that actual survey.